docs/about.html

```diff
@@ -8,7 +8,7 @@
   
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   
-  <title>About Multi-Armed Bandits &mdash; MABWiser 1.0.0 documentation</title>
+  <title>About Multi-Armed Bandits &mdash; MABWiser 1.8.1 documentation</title>
```

Comment:
ah thank you!

Reply:
Can't take credit, @dorukkilitcioglu previously pointed it out :)